### DISCUSSION TITLE
Backup

### DISCUSSION THREAD

**Discussion by vladimirs-davidovs** [Azure/ipam#198] [Q&A] [answered]
Hello! 
I'd like to know if there is a way to backup IPAM configuration as well as current reservations in any way and then, if the need be, restore those from the backup, and what would be the right way to do that?

**Comment by DCMattyG** [comment, marked as answer]
Hey @vladimirs-davidovs, as all of the data for the Azure IPAM solution is kept in Cosmos DB, I would leverage the existing backups that are built into that solution. You can configure the backups for Cosmos DB in the portal as shown here:

![image](https://github.com/Azure/ipam/assets/12955732/7c848acb-6ba7-48e4-9857-373e3934a3d4)

More details on configuring Cosmos DB backups can be found in this document:

https://learn.microsoft.com/en-us/azure/cosmos-db/periodic-backup-restore-introduction

Hope that helps!
